### DISCUSSION TITLE
Can it be used in Linux ?

### DISCUSSION THREAD

**Discussion by vlad0337187** [exodrifter/unity-python#2] [Q&A]
Hello.

Can this be used under Linux, for example, Linux Mint 18.3 ?

Thanks.

**Comment by exodrifter** [comment]
I have not tested this repository in Linux, but it should work since IronPython (the library used to execute the python code) is cross-platform.

**Comment by vlad0337187** [comment]
@exodrifter , Yes, it worked.
I misunderstood readme.
I needed to place this not to Plugins, but to Assets folder.

Also to clarify, you need to go to Edit > Project Settings > Other Settings
and there is ability to choose API version.

Thank you very much, it can be closed.

**Comment by exodrifter** [comment]
Thank you for your feedback!

I was referring to the [special folder](https://docs.unity3d.com/Manual/SpecialFolders.html), but it's true that this project does not necessarily need to be placed there. I've updated the README accordingly.

On the other hand, I'm not sure what you mean by `Edit > Project Settings > Other Settings`. This path does not exist in Unity 2017.3.0f3. Perhaps you are using a different version of Unity?

**Comment by vlad0337187** [comment]
@exodrifter , strange, but seems that when I placed it to ./Project/Plugins/,
it doesn't worked...

I'll check it another time.

**Comment by exodrifter** [comment]
The plugins folder is located at `./Project/Assets/Plugins`, not `./Project/Plugins`.

**Comment by vlad0337187** [comment]
@exodrifter , yes, right, thank you =)
